crates/ruff_python_formatter/src/statement/stmt_if.rs

```diff
@@ -36,7 +36,7 @@ impl FormatNodeRule<StmtIf> for FormatStmtIf {
                         maybe_parenthesize_expression(test, item, Parenthesize::IfBreaks),
                     ],
                 ),
-                block_indent(&body.format())
+                clause_body(body, trailing_colon_comment),
```

Comment:
We'll need to use `clause_body` in the `else` and `elif` cases too (line 101). Could we add test cases as well.